_functions/synth_parameters/delay.md

```diff
@@ -2,4 +2,5 @@
 title: delay
 category: synth_parameters
 ---
-a pattern of numbers from 0 to 1. Sets the level of the delay signal.
+a pattern of numbers that set the initial level of the delay signal.  I.e. a value of one means the first echo will be as
```

Comment:
Well, I removed it because it doesn't have to be from zero to one!  It's perfectly fine to set `delay` to greater than one, and negative values work as well, which can be handy for making a comb filter if `delaytime` is also really short.